torchgeo/datamodules/vhr10.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""NWPU VHR-10 datamodule."""
+
+from typing import Any, Callable, Dict, List, Tuple, Union
+
+import kornia.augmentation as K
+import torch
+import torchvision
+from einops import rearrange
+from torch import Generator, Tensor
+from torch.nn.modules import Module
+from torch.utils.data import random_split
+
+from ..datasets import VHR10
+from ..samplers.utils import _to_tuple
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+# kornia DataKeys must accept MASKS
+
+
+class _AugPipe(Module):
```

Comment:
Why is this needed?

Reply:
Why isn't this needed for NASA Marine Debris?

Reply:
It is needed and has been added.